qualtran/bloqs/basic_gates/z_basis.py

```diff
@@ -74,7 +76,9 @@ def __attrs_post_init__(self):
 
     @cached_property
     def signature(self) -> 'Signature':
-        return Signature([Register('q', bitsize=1, side=Side.RIGHT if self.state else Side.LEFT)])
+        return Signature(
+            [Register('q', bitsize=QBit(), side=Side.RIGHT if self.state else Side.LEFT)]
```

Comment:
this `bitsize=QBit()` uses the attrs shim to make everything work properly, yes? 

Do you want to take the opportunity to remove the `bitsize=` part of it. `bitsize=1` is much more informative than `1` but `QBit()` -- in my mind -- captures the full essence of the argument.